pkg/minikube/bootstrapper/kubeadm/kubeadm.go

```diff
@@ -226,19 +228,26 @@ func (k *Bootstrapper) init(cfg config.ClusterConfig) error {
 	conf := bsutil.KubeadmYamlPath
 	ctx, cancel := context.WithTimeout(context.Background(), initTimeoutMinutes*time.Minute)
 	defer cancel()
+	admInitLogReader, admInitLogWriter := io.Pipe()
```

Comment:
nit: use a shorter variable name for a short lived variable. Perhaps just `ar, aw`?

Reply:
Done, renamed to `kr, kw` for `kubeadm`